rskj-core/src/main/java/co/rsk/cli/CliArgs.java

```diff
@@ -92,6 +100,9 @@ public Parser(Class<O> optionsClass, Class<F> flagsClass) {
                                 throw new IllegalArgumentException("You must provide a flag name, e.g. --quiet");
                             }
                             flags.add(getFlagByName(args[i].substring(2, args[i].length())));
+                        } else if (args[i].charAt(1) == 'X' || args[i].charAt(1) == 'x') {
+                            String arg = args[i].replace("-X", "").replace("-x", "");
+                            paramValueMap.putAll(parseArgToMap(arg));
```

Comment:
Minor things, but `args[i].charAt(1)`, used three times, could be a variable like `secondChar`.
And you could do the compare like this: `secondChar.equalsIgnoreCase("X")`

In addition, maybe it would be a good idea to extract some constants for things like `X` and even `-`: they are used several times as simple chars.

Reply:
I don't think it would be efficient to use `equalsIgnoreCase` since I want to know if the chars `-` and `X` are in the these exact positions. There could be more X in other positions, so I think it's better to follow the pattern from previous checks in this case, code would be more consistent, I think I can do something to refactor a bit more by putting these chars into a variable and normalize the char to avoid replacing to times below within `parseArgToMap`